src/scripts/index.js

```diff
@@ -1 +1,89 @@
 import '../style.css';
+
+const fetch = require('node-fetch');
+
+function fillTable(data) {
+  const highestUser = document.querySelector('.highest');
+  const highestScore = document.querySelector('.highest_score');
+  highestUser.textContent = Object.values(data)[0][0].user;
+  highestScore.textContent = Object.values(data)[0][0].score;
+  const shighestUser = document.querySelector('.shighest');
+  const shighestScore = document.querySelector('.shighest_score');
+  shighestUser.textContent = Object.values(data)[0][1].user;
+  shighestScore.textContent = Object.values(data)[0][1].score;
+  const lhighestUser = document.querySelector('.lhighest');
+  const lhighestScore = document.querySelector('.lhighest_score');
+  lhighestUser.textContent = Object.values(data)[0][2].user;
+  lhighestScore.textContent = Object.values(data)[0][2].score;
+  const ehighestUser = document.querySelector('.ehighest');
+  const ehighestScore = document.querySelector('.ehighest_score');
+  ehighestUser.textContent = Object.values(data)[0][3].user;
+  ehighestScore.textContent = Object.values(data)[0][3].score;
+  const fhighestUser = document.querySelector('.fhighest');
+  const fhighestScore = document.querySelector('.fhighest_score');
+  fhighestUser.textContent = Object.values(data)[0][4].user;
+  fhighestScore.textContent = Object.values(data)[0][4].score;
+  const thighestUser = document.querySelector('.thighest');
+  const thighestScore = document.querySelector('.thighest_score');
+  thighestUser.textContent = Object.values(data)[0][5].user;
+  thighestScore.textContent = Object.values(data)[0][5].score;
+}
+
+fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
```

Comment:
- [ ] Your data/ API's are not being fetched using the `async` and the `await` methods. To ensure a correct implementation of asynchronous functions we want to fetch our API using the `async` and `await` appropriately together handling them with the `try/catch` block. Kindly ensure to improve upon this aspect in our code at **all** the places data is being fetched